### PR TITLE
Maude 2.7 [New Formula]

### PR DESCRIPTION
**Pull request by lordqwerty**
### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [x] Written a sensible test? (The best test is to compile and run a sample program.)

### Updates to existing formula: have you

- [ ] Removed the `revision` line, if any, when bumping the version number?
- [ ] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [ ] Not altered the `bottle` section?
- [ ] Checked that the tests still pass?

### CODE REVIEW

File: maude.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  homepage "http://maude.cs.illinois.edu"
+  url "http://maude.cs.illinois.edu/w/images/8/81/Maude27-osx.zip"
+  version "2.7"
+  sha256 "7b97f675ede00edb8435cee08ea41e5226eb8daf7dea7c52c7c79d70dd5fc0ff"
+
+  def install
+    bin.install "maude.darwin64" => "maude"
+    prefix.install Dir["*.maude"]
+
+    ENV["MAUDE_LIB"] = "#{HOMEBREW_PREFIX}/Cellar/#{name}/#{version}"
```
Comment (dpo) [inline maude.rb]: This won't persist. It'll only be active in the `install` method, so it doesn't seem to serve a purpose. It's good that you mention it in the `caveats` though.

Comment (lordqwerty) [inline maude.rb]: My first formula. From reading the docs didn't seem to be a better way of doing this, unless I am missing something? 

Happy to change and push an update to help get this merged quicker :) 👍

Comment (dpo) [inline maude.rb]: Just delete lines 11 and 12. Otherwise, this looks good to go.

Comment (lordqwerty) [inline maude.rb]: No it's not required for the test. Required if you are wanting to use Maude in other applications you write.

**Pull request closed without merging**